plugins/Installation/ServerFilesGenerator.php

```diff
@@ -68,7 +68,7 @@ public static function createHtAccessFiles()
             '/libs'         => $denyAll . $allowStaticAssets,
             '/vendor'       => $denyAll . $allowStaticAssets,
             '/plugins'      => $denyAll . $allowStaticAssets . $allowManifestFile,
-            '/misc/user'    => $denyAll . $allowStaticAssets,
+            '/misc'         => $denyAll . $allowStaticAssets,
```

Comment:
There is explicit .htaccess in misc/cron allowing access to the archive.php script, so this should be ok as an extended scope.